### ISSUE TITLE
Filwip does not build on x86

### ISSUE THREAD

**Issue by Vidrep** [HaikuArchives/FilWip#12]
Filwip recipe:
ARCHITECTURES="x86 x86_gcc2"

Filwip does not build on x86, but ends in an error:

FileLooper.cpp:281:26: error: 'endl' was not declared in this scope
   cout << processPath << endl; cout.flush();
                          ^
FileLooper.cpp:281:26: note: suggested alternative:
In file included from /boot/system/develop/tools/lib/gcc/i586-pc-haiku/5.4.0/include/c++/iostream:39:0,
                 from FileLooper.cpp:41:
/boot/system/develop/tools/lib/gcc/i586-pc-haiku/5.4.0/include/c++/ostream:590:5: note:   'std::endl'
     endl(basic_ostream<_CharT, _Traits>& __os)
     ^
/boot/system/develop/etc/makefile-engine:291: recipe for target 'objects.x86-cc5-release/EraserLooper.o' failed
make: *** [objects.x86-cc5-release/EraserLooper.o] Error 1
/boot/system/develop/etc/makefile-engine:291: recipe for target 'objects.x86-cc5-release/FileLooper.o' failed
make: *** [objects.x86-cc5-release/FileLooper.o] Error 1
make: Leaving directory '/sources/FilWip-e5e8bc31c3f356b603041be8883ce5c7df46ce97/Source'
[Errno -2147459069] No such file or directory: '/boot/home/haikuports/haiku-apps/filwip/work-1.0.0'
Error: Build has failed - stopping.
~/haikuports>

**Comment by humdingerb** [comment]
I improved things bit in ccff8c8da16eb9fbae21e9f7942b8fa9a3203d76

There's still this, which I have no idea how to fix:
```
PrefsWindow.cpp: In member function 'void PrefsWindow::SaveViewLoopers()':
PrefsWindow.cpp:524:89: error: cast from 'void*' to 'int8 {aka signed char}' loses precision [-fpermissive]
   (int8)lo_priorityList.ItemAt(lo_priorityPopup->IndexOf(lo_priorityPopup->FindMarked())));
                                                                                         ^
```

**Comment by humdingerb** [comment]
@korli came to the rescue. Fixed with f4a03748333960386b67c58acc9e835d4999d3a3
